tests/ecp5/logic.ys

```diff
@@ -0,0 +1,7 @@
+read_verilog logic.v
+hierarchy -top top
+equiv_opt -assert -map +/ice40/cells_sim.v synth_ice40 # equivalency check
```

Comment:
synth_ice40

Reply:
Sorry. 
Fixed all synth_ice40 to synth_ecp5; Removed *_synth.v files and generation in scripts.